cli/commands/init.js

```diff
@@ -16,18 +16,37 @@ export async function init(name = '') {
 	console.log('Initializing...');
 
 	let config = {};
+	let vesselConfig = {};
+
+	const vesselFilePath = path.join(process.cwd(), 'vessel.dhall');
+
+	if (fs.existsSync(vesselFilePath)){
+		console.log('Reading vessel.dhall file');
+		vesselConfig = await readVesselConfig()|| {};
+	}
+
+	if (vesselConfig.dependencies){
+		config.dependencies = {};
+
+		for (const dep of (vesselConfig.dependencies || [])){
+			config.dependencies[dep.name] = dep;
+		}
+	}
 
 	// lib mode
 	if (name) {
-		config = {
-			package: {
-				name: name,
-				version: '0.1.0',
-				description: '',
-				repository: '',
-			}
+		config.package = {
+			name,
+			version: '0.1.0',
+			description: '',
+			repository: '',
+			moc: vesselConfig.compiler || ''
```

Comment:
At the moment the 'moc' field is reserved but is not used anywhere, so don't expect it to work like in vessel.
It's even disabled on the backend
```
		if (config.moc.size() > 0) {
			return #err("invalid config: 'moc' field is not supported yet");
		};
```

Reply:
Thanks for the heads up. I wasn't able to publish packages when it was set.